shared/wallets/sep7-confirm/index.tsx

```diff
@@ -0,0 +1,356 @@
+              recipient={props.recipient}
+            />
+          ) : (
+            <TxInfo
+              fee={props.summary.fee}
+              memo={props.summary.memo}
+              source={props.summary.source}
+              operations={props.summary.operations}
+            />
+          )}
+        </Kb.ScrollView>
+        <Kb.Box2
+          direction="horizontal"
+          fullWidth={true}
+          centerChildren={true}
+          gap="small"
+          style={styles.buttonContainer}
+        >
+          <Kb.WaitingButton
+            type="Success"
+            onClick={props.operation === 'pay' ? () => props.onAcceptPay(props.amount) : props.onAcceptTx}
+            waitingKey={props.waitingKey}
+            fullWidth={true}
+            style={styles.button}
+            label={props.operation === 'pay' ? 'Pay' : 'Sign'}
+          />
+        </Kb.Box2>
+      </Kb.Box2>
+      <Kb.SafeAreaView />
+    </Kb.MaybePopup>
+  )
+
+const styles = Styles.styleSheetCreate({
+  bodyText: Styles.platformStyles({
+    common: {color: Styles.globalColors.black},
+    isElectron: {wordBreak: 'break-word'},
+  }),
+  button: {
+    marginBottom: Styles.globalMargins.small,
+    marginTop: Styles.globalMargins.small,
+  },
+  buttonContainer: Styles.platformStyles({
+    common: {
+      ...Styles.padding(0, Styles.globalMargins.small),
+      alignSelf: 'flex-end',
+      flexShrink: 0,
+      justifyContent: 'space-between',
+    },
+    isElectron: {
+      borderTopColor: Styles.globalColors.black_10,
+      borderTopStyle: 'solid',
+      borderTopWidth: 1,
+    },
+  }),
+  container: Styles.platformStyles({
+    common: {
+      backgroundColor: Styles.globalColors.white,
+    },
+    isElectron: {
+      height: 560,
+      width: 400,
+    },
+    isMobile: {
+      flexGrow: 1,
+      flexShrink: 1,
+      maxHeight: '100%',
+      width: '100%',
+    },
+  }),
+  dialog: {
+    padding: Styles.globalMargins.large,
+  },
+  header: Styles.platformStyles({
+    common: {
+      backgroundColor: Styles.globalColors.purpleDark,
+    },
+    isElectron: {
+      flex: 1,
+      minHeight: 160,
+    },
+    isMobile: {
+      flexBasis: 'auto',
+      flexGrow: 1,
+      flexShrink: 1,
+      minHeight: 250,
+    },
+  }),
+  headerContent: {
+    alignItems: 'center',
+    marginTop: Styles.globalMargins.tiny,
+  },
+  headingText: {
+    color: Styles.globalColors.black_50,
+    marginBottom: Styles.globalMargins.xtiny,
+  },
+  memoContainer: {
+    paddingBottom: Styles.globalMargins.tiny,
+    paddingLeft: Styles.globalMargins.small,
+    paddingRight: Styles.globalMargins.small,
+    paddingTop: Styles.globalMargins.tiny,
+  },
+  purpleText: Styles.platformStyles({
+    common: {color: Styles.globalColors.purple},
+  }),
+  scrollView: {
+    flexBasis: 'auto',
+    flexGrow: 0,
+    flexShrink: 1,
+  },
+  stellarAddressConfirmText: Styles.platformStyles({
+    isElectron: {
+      wordBreak: 'break-all',
+    },
+  }),
```

Comment:
I think `lineClamp` of `Kb.Text` works on desktop now too, so this could be replaced by that.